core/integration/module_python_test.go

```diff
@@ -1599,7 +1731,7 @@ func pythonSource(contents string) dagger.WithContainerFunc {
 }
 
 func pythonSourceAt(modPath, contents string) dagger.WithContainerFunc {
-	return fileContents(path.Join(modPath, pythonSourcePath), contents)
+	return fileContents(path.Join(modPath, "src/test/__init__.py"), contents)
```

Comment:
nit: could the name of the module become a parameter rather than hardcoding `test`? Just seems like a gotcha to have to name the module `test` in order for this to work as expected (IIUC).

Realize this might be a super tedious update, not a blocker

Reply:
I think this is fine considering the alternative. If I make it a parameter then it becomes pretty tedious whenever this is used in any language. So I figured a convention for a module named "test" is simpler. There's test cases that use a different module name, they just don't use this helper.